src/Eloquent/Builder.php

```diff
@@ -30,24 +30,24 @@ class Builder extends EloquentBuilder
         'avg',
         'count',
         'dd',
-        'doesntExist',
+        'doesntexist',
         'dump',
         'exists',
-        'getBindings',
-        'getConnection',
-        'getGrammar',
+        'getbindings',
+        'getconnection',
+        'getgrammar',
         'insert',
-        'insertGetId',
-        'insertOrIgnore',
-        'insertUsing',
+        'insertgetid',
```

Comment:
If #2659 is merged, this method will never be called. Would it be better to do away with it? Possibly by leaving a comment.
Or maybe discard #2659 and keep this PR as a fix.

Reply:
What you said in the [comment](https://github.com/mongodb/laravel-mongodb/pull/2659/files#r1379745121) in #2659 makes sense, the latter part.
Personally I like to be specific/remove magic, but that's the way Laravel works and we need to align with that.

In short, close #2659 in favour of this one.

> It seems to be fixed by [#2661](https://github.com/mongodb/laravel-mongodb/pull/2661/files#diff-5ea252bbf9d6b385257af07ce12346559d3d4e1727eef473c50b081e28e10230R40). Both solutions will work, yours removes a layer of magics but it is more fragile if the implementation changes in Laravel.